README.md

```diff
@@ -5,15 +5,20 @@ CONNECT is an open source software solution that supports health information exc
 
 This software solution was initially developed by federal agencies to support their health-related missions, but it is now available to all organizations and can be used to help set up health information exchanges and share data using nationally-recognized interoperability standards.
 
+License
+-------
+
+CONNECT is released under the: [License](https://connectopensource.atlassian.net/wiki/x/mQCD)
```

Comment:
- Remove the colon after "the"
- Change "License" to "Revised BSD License"
- Add a period at the very end